Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-juniper-isg-snmp.md

> import Tabs from '@theme/Tabs';

Reviewer: Juniper ISG SNMP?

Reviewer 2: SAAS-1970

import TabItem from '@theme/TabItem';

## Dépendances du connecteur de supervision

Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Juniper ISG** 

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer 2: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

### Pack

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md). de supervision

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.
```

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

1. Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

</Tabs>

2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Juniper ISG**

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

3. Appliquez le modèle d'hôte **Net-Juniper-Isg-SNMP-custom**.

> Si vous utilisez SNMP en version 3, vous devez configurer les paramètres spécifiques associés via la macro **SNMPEXTRAOPTIONS**.

> > Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#snmpv3-options-mapping).

Reviewer: ```suggestion
> Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#mapping-des-options-snmpv3).
```

> > Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#snmpv3-options-mapping).

Reviewer 2: ```suggestion
> Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#mapping-des-options-snmpv3).
```

<Tabs groupId="sync">
<TabItem value="Cpu" label="Cpu">

| Macro        | Description                                                                                        | Valeur par défaut | Obligatoire |
|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | WARNING      | Warning threshold in percent (1min,5min,15min)                                                     |                   |             |

Reviewer: the old text was better

> | CRITICAL     | Critical threshold in percent (1min,5min,15min)                                                    |                   |             |

Reviewer 2: ```suggestion
| WARNING      | Warning threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes.                                                      |                   |             |
| CRITICAL     | Critical threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes.                                                     |                   |             |
```
